builds/misc/packages-release.yaml

```diff
@@ -202,6 +215,14 @@ stages:
           pmcRepoName: microsoft-debian-bullseye-prod-apt
           pmcRelease: bullseye
 
+        Debian12-arm32v7:
```

Comment:
We've decided to promote all three architectures to Tier 1, so they all need to be published to PMC (and GitHub).